src/app/core/interceptors/jwt.interceptor.ts

```diff
@@ -19,6 +19,11 @@ export class JwtInterceptor implements HttpInterceptor {
   ): Observable<HttpEvent<any>> {
     // add authorization header with jwt token if available
     const currentUser = this.authenticationService.currentUserValue
+    request = request.clone({
```

Comment:
This is response form back-end to support CORS, so we don't set it add front-end

Reply:
Thanks @thienkieu , let me remove it